CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 ## 4.61.0 (Unreleased)
 
+BREAKING CHANGES:
+* cloudrunv2: set a default value of 3 for `max_retries` in `google_cloud_run_v2_job`. This should match the API's existing default, but may show a diff at plan time in limited circumstances as drift is now detected ([#14223](https://github.com/hashicorp/terraform-provider-google/pull/14223))
+
+FEATURES:
+* **New Data Source:** `firebase_android_app_config` ([#14202](https://github.com/hashicorp/terraform-provider-google/pull/14202))
```

Comment:
```suggestion
* **New Data Source:** `google_firebase_android_app_config` ([#14202](https://github.com/hashicorp/terraform-provider-google/pull/14202))
```

Reply:
Good catch! I saw one issue and not the other here